frame/vamm/src/lib.rs

```diff
@@ -313,9 +383,8 @@ pub mod pallet {
 		///
 		/// # Runtime
 		/// `O(1)`
-
 		#[transactional]
-		fn create(config: &Self::VammConfig) -> Result<Self::VammId, DispatchError> {
+		fn create(config: &VammConfigOf<T>) -> Result<VammIdOf<T>, DispatchError> {
```

Comment:
Any particular reason for this style change? A lint?

Reply:
Just consistency across function signatures in `lib.rs`.